gradle/libs.versions.toml

```diff
@@ -6,21 +6,21 @@ compileSdk = "34"
 
 # Dependencies
 kotlin = "2.0.10"
-android-gradle-plugin = "8.2.2"
+android-gradle-plugin = "8.5.2"
 coroutines = "1.8.1"
 kotlinx-datetime = "0.6.0"
-ktor = "2.3.11"
+ktor = "2.3.12"
```

Comment:
fyi, ktor `3.0.0` just released yesterday

Reply:
I wouldn't recommend upgrading to a major release before we publish Droidcon London. Who knows how many bugs there will be in the new version.

Reply:
Wasn't aware that we're publishing for Droidcon London. Makes sense

Reply:
Added a comment